src/utils/consts.ts

```diff
@@ -784,6 +784,25 @@ export const ALGORAND_HOST =
         algodServer: "http://localhost",
         algodPort: "4001",
       };
+export const ALGORAND_INDEXER =
+  CLUSTER === "mainnet"
+    ? {
+        token: "",
+        server: "https://mainnet-idx.algonode.cloud",
+        port: "443",
+      }
+    : CLUSTER === "testnet"
+    ? {
+        token: "",
+        server: "https://testnet-idx.algonode.cloud",
+        port: "443",
+      }
+    : {
```

Comment:
so far, looks good, just wondering why do we will need the localhost settings, I guess at localhost we will point to testnet or have some ENV variable to inject this value

Reply:
I did it this way to make it consistent with the way the [Algorand client is handled](https://github.com/XLabs/portal-bridge-ui/blob/db10af4ee95586a29a785711a7dabd192483cb68/src/utils/consts.ts#L781-L786).

Reply:
I think it's ok since it's following the pattern for the algorand node defined above and the default `devnet` cluster